app/models/concerns/spotlight/translatables.rb

```diff
@@ -16,7 +16,13 @@ def translates(*attr_names)
       # and defaults to the ActiveRecord value
       def define_translated_attr_reader(attr_name)
         define_method(:"#{attr_name}") do
```

Comment:
```suggestion
        # Define a dynamic method for translating database-backed attributes,
        # falling back to the database information as needed.
        #
        # Note: the empty string is provided as the final fallback to avoid i18n blowing
        # up on nil attributes.
        define_method(:"#{attr_name}") do
```